README.md

```diff
@@ -152,6 +144,26 @@ type definitions:
 - Rust: https://github.com/package-url/packageurl.rs
 - Swift: https://github.com/package-url/packageurl-swift
 
-## Users, adopters and links
+## Support
+
+If you have a specific problem, suggestion or bug, please submit a 
+[GitHub issue](https://github.com/package-url/purl-spec/issues).
+
+For quick questions or socializing, join the PURL community discussions 
+at:
+- [AboutCode Slack](https://aboutcode-org.slack.com/archives/C08LBQMA7DE)
+- [CycloneDX Slack](https://cyclonedx.slack.com/archives/C06KTE3BWEB)
+- [Gitter](https://matrix.to/#/#package-url_Lobby:gitter.im)
+
+## License
+
+Copyright (c) the purl authors
+
+The license for **purl-spec** software is MIT: 
+- SPDX-License-Identifier: MIT-License.
+- See the *LICENSE* file in this repository for the license text.
 
-See the [dedicated adopters list](ADOPTERS.md).
+The license for the ECMA-427 standard is the _ECMA Copyright License and 
+Disclaimer_.
+- SPDX-License-Identifier: LicenseRef-scancode-ecma-standard-copyright-2024
+- See page v of [ECMA-427](https://tc54.org/purl/) for the license text.
```

Comment:
IMHO, we should add this in the repo, not reference it elsewhere with a URL. Ideally the text could stored in a file named `ecma-standard-copyright-2024.LICENSE` or just stored in the same LICENSE file as the MIT text.

Or we could adopt the REUSE.software conventions?
```suggestion
```